replace/german/messages_l_german.yml

```diff
@@ -1,393 +1,393 @@
+ msg_new_theocracy_lesee_disapprove_desc:1 "[LEFT_CHARACTER.GetShortUIName|U] billigt Euch nicht #weak ([Concept('church_property', 'Kircheneigentum')|E])#!"
+ msg_mercenary_company_dismissed:0 "Söldnertrupp entlassen"
+ msg_mercenary_company_dismissed_desc:0 "Der Vertrag für [LEFT_TITLE.GetNameNoTooltip] lief aus, und die Soldaten wurden entlassen"
+ msg_holy_order_dismissed:0 "Kriegerorden entlassen"
+ msg_holy_order_dismissed_desc:0 "[LEFT_TITLE.GetNameNoTooltip] wurde entlassen"
+ msg_fired_from_council:0 "Aus dem Rat geworfen"
+ msg_fired_from_council_desc:0 "[LEFT_CHARACTER.GetShortUIName|U] hat Euch aus dem [council|E] geworfen"
+ msg_council_swap_position:0 "Neuer Ratsposten"
+ msg_council_swap_position_desc:2 "Ihr seid jetzt $POSITION_NAME$ im [council|E] von [LEFT_CHARACTER.GetShortUIName]"
+ msg_left_council_position:0 "Ratsmitglied gegangen"
+ msg_left_council_position_desc:0 "[LEFT_CHARACTER.GetShortUIName|U] hat den Posten als $POSITION_NAME$ aufgegeben"
+ msg_peace_armies_disbanded:0 "Im Frieden"
+ msg_peace_armies_disbanded_desc:1 "Ihr habt endlich Frieden, weshalb alle Eure Armeen aufgelöst wurden"
+ msg_invalidate_war_title:1 "[war.GetName] vorbei: Es war nicht mehr möglich, ihn fortzusetzen."
+ msg_claim_war_invalidated_message:1 "[war.GetName] vorbei: [DEFENDER.GetShortUINameU] kontrolliert keine der beanspruchten Gemeinden mehr."
+ msg_claim_war_invalidated_claimant_unavailable_dead_message:1 "[war.GetName] ist vorbei: [claimant.GetShortUIName] ist gestorben, und daher kann [claimant.Custom('DE_PRO_POS_S_3P')] Anspruch nicht mehr von [attacker.GetShortUIName] eingesetzt werden."
+ msg_claim_war_invalidated_claimant_unavailable_not_vassal_message:1 "[war.GetName] ist vorbei: [claimant.GetShortUIName] ist kein Vasall oder Höfling von [ATTACKER.GetShortUINameU] und daher kann [claimant.Custom('DE_PRO_POS_S_3P')] Anspruch nicht mehr eingesetzt werden."
+ msg_de_jure_duchy_war_invalidated_message:1 "[war.GetName] vorbei: [defender.GetShortUINameU] kontrolliert kein Land mehr im umkämpften Herzogtum."
+ msg_de_jure_county_war_invalidated_message:1 "[war.GetName] vorbei: [defender.GetShortUINameU] kontrolliert die Gemeinde nicht mehr."
+ msg_de_jure_duchy_war_invalidated_liege_war_message:1 "Ein Krieg hat geendet ([war.GetName]): [Select_CString(attacker.IsLocalPlayer, 'Ihr', attacker.GetShortUIName)] [Select_CString(attacker.IsLocalPlayer, 'seid', 'ist')] dem Krieg des eigenen Lehnsherrn gegen [Select_CString(defender.IsLocalPlayer, 'Euch', defender.GetShortUIName)] beigetreten."
+ msg_de_jure_all_war_invalidated_message:1 "[war.GetName] vorbei: [defender.GetShortUINameU] kontrolliert kein de jure beanspruchtes Land mehr."
+ msg_reclaim_war_invalidated_message:1 "[war.GetName] vorbei: [defender.GetShortUINameU] kontrolliert kein de jure beanspruchtes Land der Nachtwache mehr."  #Selbst geschrieben/abgetippt
```

Comment:
Wird De Jure in dem Fall wirklich klein geschrieben? Bin mir nicht sicher, wegen der Vanilla_l @Dillo234